.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Question
+about: Ask a question about using the software
+title: "[Question] "
+labels: ["question", "to confirm"]
```

Comment:
```suggestion
labels: ["question"]
```

I think "to confirm" is not useful for a question

Reply:
Of course!